docs/section21/Rule21-10.md

```diff
@@ -0,0 +1,70 @@
+
+# Boiler - Rule 21-10  
+
+**Rule ID:** 21-10  
+**Rule Description:** When the building uses boilers, the hot water pump shall be modeled as riding the pump curve if the hot water system serves less than 120,000 ft^2 otherwise it shall be modeled with a VFD.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.5 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. P-RMR is not modeled with purchased heating.
```

Comment:
This rule applies irrespective of purchased heating. Suggest to remove this applicability check.

Reply:
as discussed during the RDS call, 21-10 will be updated to include PHW, b systems and c systems will be added to the applicability check